static/sass/styles.scss

```diff
@@ -294,9 +288,28 @@ html {
   display: inline-block;
 }
 
-.middle-page-tabs {
-  @media screen and (min-width: $breakpoint-large) {
-    padding-left: 20.3rem;
+.p-tabs--25-75 {
+  @extend %vf-row;
+  @extend %vf-pseudo-border--bottom;
+  position: relative;
+  margin: 0 auto $spv--x-large;
+
+  .p-tabs__list {
+    margin-bottom: 0;
+
+    @media screen and (min-width: $breakpoint-large) {
```

Comment:
This is the same media query as in line 309, so I guess they could be merged together.